### DISCUSSION TITLE
OpenJdk 11 LTS Support

### DISCUSSION THREAD

**Discussion by mmmiich** [adoptium/adoptium-support#795] [Q&A] [answered]
Hello,

I would ask about Temurin Java 11 (LTS) support. On https://adoptium.net/support/ its stated that End of availability is set to "At least Oct 2024". Does it mean it will be supported with newest updates provided by Oracle extended support?  Or will it only contains free public updates (which ends in september 2023)? 

Best regards

**Comment by karianna** [comment, marked as answer]
free public updates (which ends in September 2023)

Eclipse Temurin binaries are a down stream distribution of OpenJDK, not Oracle's JDK.
